tests/filecheck/transforms/convert_memref_args_to_ptr.mlir

```diff
@@ -0,0 +1,42 @@
+// RUN: xdsl-opt -p convert-memref-to-ptr{lower_func=true} --split-input-file --verify-diagnostics %s | filecheck %s
+
+// CHECK:       builtin.module {
```

Comment:
the `builtin.module `is implied if omitted, saving a few more lines that do not need to be checked/matched.

Reply:
I actually recommended it here, as it makes it clear that there's no extra things inserted.